fetch.bs

```diff
@@ -6841,14 +6799,14 @@ partial interface mixin WindowOrWorkerGlobalScope {
 
 <p>The
 <dfn id=dom-global-fetch method for=WindowOrWorkerGlobalScope><code>fetch(<var>input</var>, <var>init</var>)</code></dfn>
-method, must run these steps:
+method steps are:
```

Comment:
The object construction in here is pretty sketchy, but I imagine you're aware already, and any solutions would probably be larger diffs than makes sense for this PR.

Reply:
Yeah, there's a number of places where we don't specify realms and I basically need to write tests. But I also want that to be separate as it's no longer editorial.